src/app/clusters/camera-av-settings-user-level-management-server/camera-av-settings-user-level-management-server.h

```diff
@@ -169,20 +169,21 @@ class CameraAvSettingsUserLevelMgmtServer : public AttributeAccessInterface, pub
      * digital movement. This should be invoked by a delegate on the conclusion of allocating a video stream via the AV Stream
      * Management cluster.
      */
-    void AddMoveCapableVideoStream(uint16_t aVideoStreamID, Structs::ViewportStruct::Type aViewport);
+    void AddMoveCapableVideoStream(uint16_t aVideoStreamID, chip::app::Clusters::Globals::Structs::ViewportStruct::Type aViewport);
```

Comment:
Again, no need for the `chip::app::Clusters::` bit.  And similar elsewhere.

Reply:
Fixed